src/VisualStudio/IntegrationTest/TestUtilities/InProcess/Debugger_InProc.cs

```diff
@@ -33,5 +56,105 @@ public void SetBreakPoint(string fileName, int lineNumber, int columnIndex)
         public void ExecuteStatement(string statement) => _debugger.ExecuteStatement(statement);
 
         public Common.Expression GetExpression(string expressionText) => new Common.Expression(_debugger.GetExpression(expressionText));
+
+        /// <summary>
+        /// Executes the specified action delegate and retries if Operation Not Supported is thrown.
+        /// </summary>
+        /// <param name="action">Action delegate to exectute.</param>
+        private void WaitForRaiseDebuggerDteCommand(Action action)
+        {
+            bool actionSucceeded = false;
+
+            Func<bool> predict = delegate
```

Comment:
>predict [](start = 23, length = 7)

typo

Reply:
Thanks!

---
In reply to: [136207212](https://github.com/dotnet/roslyn/pull/21824#discussion_r136207212) [](ancestors = 136207212)